apps/consent/services/galoy-auth/auth-instance.ts

```diff
@@ -2,8 +2,25 @@ import { env } from "../../env";
 import axios, { AxiosInstance } from "axios";
 
 const axiosInstance: AxiosInstance = axios.create({
-    baseURL: env.CORE_AUTH_URL,
-    headers: { "Content-Type": "application/json" },
+  baseURL: env.CORE_AUTH_URL,
+  headers: { "Content-Type": "application/json" },
 });
 
+axiosInstance.interceptors.request.use(
+  (config) => {
+    const xRealIp = config.headers["x-real-ip"];
+    const xForwardedFor = config.headers["x-forwarded-for"];
+    if (xRealIp) {
+      config.headers["x-real-ip"] = xRealIp;
+    }
+    if (xForwardedFor) {
+      config.headers["x-forwarded-for"] = xForwardedFor;
+    }
```

Comment:
this feels like a no-op? we' taking a value from `config.headers["x-real-ip"]` and redefining to the same value? same for `config.headers["x-forwarded-for"]`?

Reply:
yup this will not work need to add config

Reply:
if you do `axiosInstance.interceptors.request.use(config => config)` this doesn't work?

Reply:
no